lib/nextSafe.js

```diff
@@ -3,7 +3,16 @@ const buildCSPHeaders = require('./buildCSPHeaders.js')
 const buildPermissionsPolicyHeaders = require('./buildPermissionsPolicyHeaders.js')
 
 
+function hObj(key, value, defaultValue) {
```

Comment:
Can we give this a more expressive name? I'm all about abstracting functions to remove duplication, but I want to make sure that those functions are clearly names.

Reply:
says the person who named the CSP directive resolver `v`